### PR TITLE
Umbrella crate: exclude chain-specific crates

### PR DESCRIPTION
**Pull request by ggwpez**
Uses custom metadata to exclude chain-specific crates.  
The only concern is that devs who want to use chain-specific crates, still need to select matching versions numbers. Could possibly be addresses with chain-specific umbrella crates, but currently it should be possible to use [psvm](https://github.com/paritytech/psvm).

### CODE REVIEW

File: bridges/chains/chain-polkadot-bulletin/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,9 @@ edition.workspace = true
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 repository.workspace = true
 
+[package.metadata.polkadot-sdk]
+exclude-from-umbrella = true
```
[the remark this answers is not in the document]
Comment (ggwpez) [reply bridges/chains/chain-polkadot-bulletin/Cargo.toml:11]: Nope. I looked for a spot to document it, but we dont have a `crates.md` or something... also the things about docs.rs target could be explained there.

[the remark this answers is not in the document]
Comment (bkchr) [reply bridges/chains/chain-polkadot-bulletin/Cargo.toml:11]: CONTRIBUTION.md?

[the remark this answers is not in the document]
Comment (ggwpez) [reply bridges/chains/chain-polkadot-bulletin/Cargo.toml:11]: Yea will put it in there now.

[the remark this answers is not in the document]
Comment (ggwpez) [reply bridges/chains/chain-polkadot-bulletin/Cargo.toml:11]: done

**Review by bkchr — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Review by lexnv — APPROVED** [review]

**Pull request merged**